src/argref/main.py

```diff
@@ -52,7 +132,10 @@ def on_page_markdown(self, markdown, **kwargs):
         """
         for autolink in self.config["autolinks"]:
             markdown = replace_autolink_references(
-                markdown, autolink["reference_prefix"], autolink["target_url"]
+                markdown,
+                autolink["reference_prefix"],
+                autolink["target_url"],
+                autolink.get("filter_links", True) is True,
```

Comment:
I think this should default to False - primarily for backwards compatibility, but also because users should opt into more complex behavior rather than opt out of it.

Reply:
Agreed, will be adjusted